plugins/woocommerce/includes/class-wc-countries.php

```diff
@@ -1286,6 +1286,16 @@ public function get_country_locale() {
 							'priority' => 69,
 						),
 					),
+					'KN' => array(
+						'postcode' => array(
+							'required' => false,
+							'label' => __( 'Postal code', 'woocommerce' ),
```

Comment:
The spacing on the `=>` needs to align with the `=>` on the previous line on both `label` properties added here

Reply:
Thank you @rrennick, changes submitted